docs/articles/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.md

```diff
@@ -159,6 +228,20 @@ Method | Description
 `parent(cssSelector)` | Finds all parents of all nodes in the matching set and filters them by `cssSelector`.
 `parent(filterFn, dependencies)` | Finds all parents of all nodes in the matching set and filters them by the `filterFn` predicate. Use an optional `dependencies` parameter to pass functions, variables or objects to the `filterFn` function. See [Filtering DOM Elements by Predicates](#filtering-dom-elements-by-predicates).
 
+```js
+// Selects all ancestors of all ul elements.
+Selector('ul').parent();
```

Comment:
In fact, we get the first ul and its first parent. Although we use "all parents" in the description of method, here we describe the specific element selector. I think for others examples it should be clarified as well.

Reply:
I disagree, this selector will work strictly in described way. In all examples above we say: `Selects all elements which ....`, because nowhere we use `nth(0)`. 
Probably we should have little topic with information about difference between:
`const allDivElements = Selector('div')` and `const firstDivElement = await Selector('div')`, I've already faced with questions about it. This conception is not obvious for some users because requires deep level of Selector understanding.